main/reference/ertp-api/brand.md

```diff
@@ -1,76 +1,79 @@
-# Brand
-A `brand` identifies the type of assets. A particular `brand` has one-to-one relationships
-with an `issuer` and a `mint`, and one-to-many relationships with `purses` and `payments`.
+# Brand Object
+A **Brand** identifies the asset type of the **[Issuer](./issuer.md)** and **[Mint](./mint.md)** associated with the **Brand**. A given **Brand** has a one-to-one relationship
+with an **Issuer** and a **Mint**, and a zero-to-many relationship with **[Purses](./purse.md)** and **[Payments](./payment.md)**.
```

Comment:
A Brand/Mint/Issuer triad doesn't necessarily have an attached Purse, correct? After all, creating a Purse is a separate method call, and you could, in theory, not ever create a Purse for the triad, yes? Granted, the triad needs a Purse in order to be useful, because Mint needs a container in which to put newly minted assets. Still, it seems like "zero-to-many" is technically a more accurate representation of the relationship, isn't it?

Reply:
one-to-one, one-to-many, and many-to-many are [technical terms](https://en.wikipedia.org/wiki/One-to-many_(data_model)) from databases or data modeling.  it's not a count of how many objects might exist, as it sounds like. 

The question it answers is whether each object that exists is associated with exactly one, or possibly more than one of the other object. If there is a purse, it's associated with exactly one Issuer/Brand/Mint.  The *many* side can vary from zero to large, but there has to be at least one object in order to ask what the relationship is.